dbt/adapters/bigquery/python_submissions.py

```diff
@@ -43,7 +45,7 @@ def __init__(self, parsed_model: Dict, credential: BigQueryCredentials) -> None:
         self.timeout = self.parsed_model["config"].get(
             "timeout", self.credential.job_execution_timeout_seconds or 60 * 60 * 24
         )
-        self.retry = retry.Retry(maximum=10.0, deadline=self.timeout)
+        self.retry = retry.Retry(predicate=POLLING_PREDICATE, maximum=10.0, timeout=self.timeout)
```

Comment:
According to https://googleapis.dev/python/google-api-core/latest/_modules/google/api_core/future/polling.html
`predicate=POLLING_PREDICATE,` is already a default so you can skip it

```suggestion
        self.retry = retry.Retry(maximum=10.0, timeout=self.timeout)
```

Reply:
My read on this is that if we supply a custom `Retry` to polling (i.e. that replaces the `DEFAULT_POLLING` retry) it won't necessarily add the default polling predicate

Reply:
Yeah I guess you're right, the `DEFAULT_POLLING` is not used anymore then and [the retry constructor doesn't have a predicate by default](https://googleapis.dev/python/google-api-core/latest/retry.html)